core/src/main/tut/README.md

```diff
@@ -88,6 +88,7 @@ libraryDependencies ++= Seq(
 For a full example of `build.sbt` you can have a look at this [build.sbt](https://github.com/melrief/pureconfig/blob/master/example/build.sbt)
 used for the example.
 
+Earlier versions of Scala had bugs which can cause subtle compile-time problems in PureConfig. As a result we recommend only using 2.11.9 or 2.12.1 or newer within the minor series.
```

Comment:
2.11.9 is a "forbidden" version :) Both 2.11.9 and 2.11.10 were intended to be the last version in the 2.11.x series, but they both introduced binary compatibility issues and were quickly superseded by 2.11.11. for that reason, I think we should recommend using 2.11.11.

See https://github.com/scala/scala/releases/tag/v2.11.11 for more details.